common/config/config.py

```diff
@@ -169,7 +170,7 @@ class Config:
     # Testing settings
     fuzz_fail_pct_name: Final[str] = "FUZZ_FAIL_PCT"
 
-    _pg_null_value: Final[object] = object()
+    _pg_null_value: Final[str] = ""  # to have a uniform (str) type returned by pg_* properties.
```

Comment:
1. Sorry, I don't understand how it makes logic different from assigning it to empty string? We currently still return `self._pg_null_value` from pg_* properties (the same exact object), so in both cases (whether it's an empty string or the same exact `object()`) it seems to work the same.  
2. Also, the only reason I made an adjustment is because test_hide_sensitive fails:
"TypeError: object of type 'object' has no len()", thrown from `sensitive_info_list property`
3. Lastly, the pg_* properties returns str and `config/utils.py` seems to handle only str and list and is not aware that that default for pg_* properties can be object.

What should I do with it? I can try to reconcile it, if you want me to do it.

Reply:
Discussed f2f - reverted these changes. Fixed the unit test (`test_hide_sensitive`) with the default parameters when invoked directly (and not through the `deploy_check`)